R/insert.R

```diff
@@ -2,14 +2,30 @@
 #' @param pkgs list of packages to insert
 #' @param repository repository folder to insert
 #' @param archive whether to archive the packages after insertion to prevent duplicates
+#' @param OSflavour optional OS flavour specification, defaults to NULL unless detected as aarch64
 #' @param ... parameters passed to `drat::archivePackages`
 #' @details
 #' repository folder should most likely correspond to the name
 #' of the repository set in the pkg metadata, however
 #' realistically, the time that they need to match when
 #' repos is set in a user session in the `getOption('repos')` field
 #' @export
-insert_packages <- function(pkgs, repository, archive = TRUE, ...) {
+insert_packages <- function(pkgs, repository, archive = TRUE, OSflavour = NULL, ...) {
```

Comment:
since this will be pretty uncommon to need to set, lets actually shift it to _after_ the dots, so the dots take precedence